packages/next/src/build/webpack-config.ts

```diff
@@ -538,12 +539,13 @@ export default async function getBaseWebpackConfig(
   ].filter(Boolean)
 
   const middlewareLayerLoaders = [
+    'next-flight-loader',
```

Comment:
Ideally they should turn to sth empty like client reference since you cannot run any server actions in middleware, should be fine for now. later we can transform client and server references into an empty object proxy in middleware and instrumentation.